tests/Build/Changelog/ChangelogBuilderTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace Aws\Test\Build\Changelog;
+
+use Aws\Build\Changelog\ChangelogBuilder;
+
+class ChangelogBuilderTest extends \PHPUnit_Framework_TestCase
+{
+	/**
+	 * @covers ChangelogBuilder
+	 *
+	 */
+
+	private $RESOURCE_DIR= "tests/Build/Changelog/resources";
+
+	private function getChangelogBuilder()
+	{
+		return new ChangelogBuilder();
+	}
+
+	public function testreadChangelogNoDirectory()
+	{
+		$this->setExpectedException(\Exception::class);
```

Comment:
I would recommend using a more granular exception so that you don't accidentally accept an unrelated exception as a test success. I also find the `@expectedException` annotation more clear than calling `$this->setExpectedException`, but that's an optional change.